prow/plugins/milestonestatus/milestonestatus.go

```diff
@@ -55,9 +56,17 @@ func init() {
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	pluginHelp := &pluginhelp.PluginHelp{
 		Description: "The milestonestatus plugin allows members of the milestone maintainers Github team to specify the 'status/*' label that should apply to a pull request.",
-		Config: map[string]string{
-			"": fmt.Sprintf("The milestone maintainers team is the Github team with ID: %d.", config.Milestone.MaintainersID),
-		},
+		Config: func(repos []string) map[string]string {
```

Comment:
nit: This is probably a little easier to read without using a function here. Just create and populate `configMap` before defining `pluginHelp`.
Its a neat way to inline logic in the struct though 😄so I say keep it. (You don't need to pass the `repos` field, `enabledRepos` can just be captured like `config` is.)